### ISSUE TITLE
Not working with the new TBC prepatch

### ISSUE THREAD

**Issue by ghostlyruder** [prowley/SteaSummon#60]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by kzieres61** [comment]
Can Confirm that SteaSummon does not work under the new TBC patch for Classic wow.

**Comment by prowley** [comment]
I see much less value in SteaSummon for tbc, but I'm willing to be convinced otherwise. I am looking for a classic-classic maintainer, since I won't be playing that.

**Comment by ghostlyruder** [comment]
Summons will be needed for sure a lot will want summons to get crafting cd. I would love if you fix it

**Comment by kzieres61** [comment]
Sure. Tbc has summoning stones that can be used. But honestly I really
enjoyed having the list of ppl needing a summon.
The auto announcement.
In my guilld i am known for charging 10000 gold for summons :)
It may not be needed, but it would surely be nice to Have again!

**Comment by ghostlyruder** [comment]
I would love to have it

**Comment by axlroden** [comment]
raidsummon has updated for TBC, which I would consider enough for TBC. 
Steasummon is needed in Classic ..

**Comment by kzieres61** [comment]
Yes. But steasummon works much better than raid summon in my opinion.
The list is better. It doesn't just clear when a summon is done. The name
clears when accepted.
It has an announce feature so u dont have to type out the announcement.
It us just a better version overall.

**Comment by axlroden** [comment]
of course.. but hey.. how much you used a raid summon in general since tbc?
